Formula/audacious.rb

```diff
@@ -65,31 +66,26 @@ class Audacious < Formula
   fails_with gcc: "5"
 
   def install
-    args = std_meson_args + %w[
+    args = %w[
       -Dgtk=false
-      -Dqt=true
+      -Dqt6=true
     ]
 
-    mkdir "build" do
-      system "meson", *args, "-Ddbus=false", ".."
-      system "ninja", "-v"
-      system "ninja", "install", "-v"
-    end
+    system "meson", "setup", "build", *std_meson_args, *args, "-Ddbus=false"
+    system "meson", "compile", "-C", "build", "--verbose"
+    system "meson", "install", "-C", "build"
 
     resource("plugins").stage do
       args += %w[
         -Dcoreaudio=false
```

Comment:
Are the caveats about Core Audio still true?

https://github.com/Homebrew/homebrew-core/blob/06cf0f473644c1cd07e256995788330a28a6f031/Formula/audacious.rb#L99

This seems to originate from a _long_ time ago. https://github.com/Homebrew/homebrew-core/commit/bf7e79ef20557e25be2c13399da2c4b4683b4eca

Reply:
Let me cherry-pick #126204.